src/setup/templates/rgw.logrotate.tmpl

```diff
@@ -0,0 +1,13 @@
+TMP_LOG_PATH/*.log
+{
+    rotate 7
+    maxsize 50M
+    weekly
+    compress
```

Comment:
what impact this 'compress' has, so once file reaches its limit(50MB here), it will be compressed to what size, basically we need the compression ratio, so we can decide size of rgw logs in support bundle

Reply:
compress will compress the log files with gzip utility.
so, as explored earlier, compression ratio with gzip will be ~80%

Reply:
yeah i read, by default its compressing with gzip with best compression (-9 option)